temporal-sdk/src/test/java/io/temporal/workflow/shared/TestWorkflowWithCronScheduleImpl.java

```diff
@@ -46,15 +49,16 @@ public String execute(String testName) {
       return null;
     }
 
-    lastCompletionResult = Workflow.getLastCompletionResult(String.class);
     lastFail = Workflow.getPreviousRunFailure();
+    retryCount.computeIfAbsent(testName, k -> new AtomicInteger());
+    lastCompletionResults.computeIfAbsent(testName, k -> new HashMap<>());
 
-    AtomicInteger count = retryCount.get(testName);
-    if (count == null) {
-      count = new AtomicInteger();
-      retryCount.put(testName, count);
+    int c = retryCount.get(testName).incrementAndGet();
```

Comment:
computeIfAbsent() returns the value. Use it instead of the second get.

Also, please use meaningful names than `c`.